utils/subprocess_output.py

```diff
@@ -8,23 +9,29 @@
 
 
 # FIXME: python 2.7 has a far better way to do this
-def get_subprocess_output(command, log):
+def get_subprocess_output(command, log, shell=None, stdin=None):
```

Comment:
Probably has no impact at all, but the default value of `shell` in `subprocess.Popen` is `False`, not `None` (see https://docs.python.org/2/library/subprocess.html#subprocess.Popen).